src/emu-i386/simx86/sigsegv.c

```diff
@@ -338,7 +317,7 @@ int e_emu_fault(struct sigcontext *scp)
    *	TheCPU.err.
    *    For page faults the current PC is recovered from the tree.
    */
-  if (!CONFIG_CPUSIM) {
+	if (_trapno!=0x0e && _trapno != 0x00) return 0;
```

Comment:
To catch divisions by zero in JIT compiled code (see further down in sigsegv.c).